src/core/js/views/notifyPushView.js

```diff
@@ -1,124 +1,122 @@
-define([
-  'core/js/adapt'
-], function (Adapt) {
+import Adapt from 'core/js/adapt';
```

Comment:
⚠️ With this PR I can no longer seem to trigger push notifications e.g.
```js
require('core/js/adapt').notify.push({ title: 'Hello world' });
```

Reply:
`Collection.prototype.model` can't be a function, apparently.